TLM/TLM/Lifecycle/Patcher.cs

```diff
@@ -44,6 +45,7 @@ public static class Patcher {
 #endif
             AssertCitiesHarmonyInstalled();
             fail = !PatchAll(HARMONY_ID, forbidden: typeof(CustomPathFindPatchAttribute));
+            fail = fail || !PatchManual(HARMONY_ID);
```

Comment:
Would this be more legible as:

```csharp
fail |= !PatchManual(HARMONY_ID);
```
?

Reply:
This one is just personal preference. We can change it to match TMPE'S style

Reply:
It's a huge codebase - every little bit helps :)

Reply:
Fixed in https://github.com/CitiesSkylinesMods/TMPE/pull/1386/commits/558c9a8bb370d5c45046e0ff1c95a3ab7d9cfa60